src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -427,6 +427,40 @@ sl_status_t SetWifiConfigurations()
     return status;
 }
 
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+/**
+ * @brief Converts the Matter Power Save Configuration to the SiWx Power Save Configuration
+ *
+ * @param configuration Matter Power Save Configuration
+ *
+ * @return sl_si91x_performance_profile_t SiWx Power Save Configuration; Default value is High Performance
+ *                                        kHighPerformance: HIGH_PERFORMANCE
+ *                                        kConnectedSleep: ASSOCIATED_POWER_SAVE
+ *                                        kDeepSleep: DEEP_SLEEP_WITH_RAM_RETENTION
+ */
```

Comment:
should the brief be in the header

Reply:
Function isn't defined in the header since it doesn't require any members of the class. 
Reduces the API surface and if a change is limited to the function doesn't recompile the full dependency tree